server/resp/src/test/java/org/infinispan/server/resp/dist/StringCommandsClusteredTest.java

```diff
@@ -38,4 +46,21 @@ protected void amendConfiguration(ConfigurationBuilder configurationBuilder) {
    protected TestSetup setup() {
       return TestSetup.clusteredTestSetup(3);
    }
+
+   @Test
+   public void testMget() {
```

Comment:
Is the override needed?

Reply:
nope, I copied it to debug in IDE.
cleaned up thanks